ci/scripts/run-fpga-cw310-tests.sh

```diff
@@ -11,6 +11,14 @@ set -e
 SHA=$(git rev-parse HEAD)
 readonly SHA
 
+if [ $# == 0 ]; then
```

Comment:
When this is merged in, PRs will need to be rebased onto it, which we should announce.

Reply:
just curious: do you know if there is a way we can perform a mass close & reopen?

Reply:
I was trying to make a joke about rebasing and pushing from within this shell script but translating from my aliases is too much work to be worht it. It would be nice to be able to force people to rebase for changes like this.

Reply:
As an alternative, we can increase the timeout to prevent the failure, and use the old behavior as the default here.